### PR TITLE
[bridge] Bridge 2024

### PR DESCRIPTION
**Pull request by damirka**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-framework/packages/bridge/sources/btc.move [hunk 1]
```diff
@@ -31,11 +27,6 @@ module bridge::btc {
...
 
-    public fun decimal(): u8 {
-        DECIMAL
-    }
-
-    public fun multiplier(): u64 {
-        MULTIPLIER
-    }
+    public fun decimal(): u8 { DECIMAL }
+    public fun multiplier(): u64 { MULTIPLIER }
```
Comment (tnowacki) [inline crates/sui-framework/packages/bridge/sources/btc.move:31]: nit, I'd probably keep this PR limited to '24 changes, but aint my code

File: crates/sui-framework/packages/bridge/sources/limiter.move [hunk 2]
```diff
@@ -137,14 +142,15 @@ module bridge::limiter {
...
-        let route_limit = option::destroy_some(route_limit);
+        let route_limit = self.transfer_limits.try_get(&route);
+        assert!(route_limit.is_some(), ELimitNotFoundForRoute);
+        let route_limit = route_limit.destroy_some();
         let route_limit_adjusted = (route_limit as u128) * (treasury::decimal_multiplier<T>() as u128);
 
         // Compute notional amount
         // Upcast to u128 to prevent overflow, to not miss out on small amounts.
-        let notional_amount_with_token_multiplier = (*vec_map::get(&self.notional_values, &treasury::token_id<T>()) as u128) * (amount as u128);
+        let value = (self.notional_values[&treasury::token_id<T>()] as u128);
```
Comment (tnowacki) [inline crates/sui-framework/packages/bridge/sources/limiter.move:152]: soon

**Review by tnowacki — COMMENTED** [review]
Otherwise looks good! 
But not approving since I'm just here because it was linked in the meeting :)

**Comment by patrickkuo** [comment]
Thanks @damirka , let’s don’t merge this just yet, there are some changes with how we create new coins, which will need to be in for audit by Friday :| will be good if we can hold this for now.

**Comment by dariorussi** [comment]
> Thanks @damirka , let’s don’t merge this just yet, there are some changes with how we create new coins, which will need to be in for audit by Friday :| will be good if we can hold this for now.

@patrickkuo out of curiosity why do we want to keep it out of auditing?

@damirka can you rebase and see if all tests pass?

**Pull request merged**